modules/editor/inserter/style.scss

```diff
@@ -0,0 +1,136 @@
+.inserter__button {
+	display: inline-block;
+	position: relative;
+	background: none;
+	border: none;
+	padding: 0;
+	margin-top: 20px;
+
+	a {
+		display: inline-block;
+		color: #87919d;
+		cursor: pointer;
+		border-radius: 12px;
+		border: 2px solid #87919d;
+		width: 24px;
+		height: 24px;
+		padding: 0;
+		margin: 0;
+
+		.dashicons {
+			padding: 1px 0;
+		}
+
+		&:hover {
+			color: #12181e;
+			border-color: #12181e;
+		}
+	}
+}
+
+.inserter {
+	width: 280px;
+	box-shadow: 0px 3px 20px rgba( 18, 24, 30, .1 ), 0px 1px 3px rgba( 18, 24, 30, .1 );
+	border: 1px solid #e0e5e9;
+	position: absolute;
+	left: -130px;
+	bottom: 40px;
+	background: #fff;
+
+	.inserter__arrow {
```

Comment:
Also let's avoid nesting as much as possible when we have a class to keep specificity at the minimum.